<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mMemberPurchases</class>
 <widget class="QWidget" name="mMemberPurchases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="membershipPurchaseTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>711</width>
     <height>561</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>481</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/poke.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="memberIDcomboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>45</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>47</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Member ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="grandTotalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Grand Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="grandTotalLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>255</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>115</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>116</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemPurchasedLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>152</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Item purchased</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="salePriceLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>185</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>186</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sale price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantityLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>222</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="orderIDcomboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Order ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="memberNameComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MemberIDlabel_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Member Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Reset Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>* Please use the &quot;Reset Search&quot; button to switch search by member name or ID!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;</string>
   </property>
   <property name="text">
    <string>Membership Purchases</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
